db/re/item_group_db.yml

```diff
@@ -29368,3 +29411,87 @@ Body:
           - Item: Up_Demon_Slayer_Shot
             RefineMinimum: 7
             RefineMaximum: 7
+  - Group: MONTHLY_PACKAGE_1
```

Comment:
If possible please put a link to your source of information above each group.

Reply:
Source links have been added to Random Option Group entries. Other data is based on client files.